### DISCUSSION TITLE
Tracking pixel in Support Ukraine site

### DISCUSSION THREAD

**Discussion by uazo** [bromite/bromite#2466] [General]
Carl, you know, I agree.
only one thing: that site tracks users with a pixel, but gives no explanation of what it is for (or at least I did not find it).
perhaps it would be better if the link pointed to the repo https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md

**Comment by csagan5** [comment]
> that site tracks users with a pixel, but gives no explanation of what it is for (or at least I did not find it).

You could open an issue there and ask about it.
This is what it does:
```
window.onload = function() {
    var e = encodeURIComponent,
        t = document,
        n = window,
        a = window.screen,
        o = navigator,
        r = n.devicePixelRatio,
        i = (new Date).getTimezoneOffset();
    try {
        i = Intl.DateTimeFormat().resolvedOptions().timeZone
    } catch (e) {}
    var s = t.createElement("img");
    s.height = s.width = "1", s.style = "border-style:none;", s.src = "https://vsh.pp.ua/pixel?page=" + e(n.location.pathname) + "&ref=" + e(t.referrer) + "&scr=" + parseInt(a.width * r) + "x" + parseInt(a.height * r) + "&lng=" + (o.language || o.browserLanguage) + "&tz=" + i
};
```

The official site to link would be https://war.ukraine.ua/support-ukraine/ but it has resources from 3 tracker domains.

> perhaps it would be better if the link pointed to the repo https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md

GitHub itself has many more; and all of the linked sites on that page also have trackers; some links from the README might also have trackers on their pages.

The Bromite official website will never have any tracker, but I do not commit to never have any link to any site with trackers.

**Reply by uazo** [reply]
> This is what it does:

it is not so much that but the cookie that installs

> You could open an issue there and ask about it.

there was already https://github.com/vshymanskyy/StandWithUkraine/issues/44 I think

> GitHub itself has many more

of course, but the bromite repo is on github

> but I do not commit to never have any link to any site with trackers.

I just wanted to bring it to your attention

**Comment by vshymanskyy** [comment]
Hey everyone. The tracking pixel currently collects (mostly) anonymized/unlinked statistics.
The whole database is basically a single JSON file: http://vsh.pp.ua/stats.json
If it collects too much, I can make it collect less data.
The tracking pixel was implemented over the weekend because I didn't want to collect data via google analytics o other tools.

**Comment by DI555** [comment]
imho, the devs should be politics neutral, and do not place any politics stuff on the main page..
and even for that - if it‘s tracking one stuff((
so, even more, it may sounds very unnormal and weird, - building track-free browser and place on the main page tracking stuff((
